libs/express/status/src/controllers/express-status.controller.ts

```diff
@@ -24,14 +24,14 @@ export class ExpressStatusController {
 	public get(req: Request, res: Response): Response {
 		return res.status(HttpCodes.OK).json({
 			project: {
-				name: Env.get('NAME'),
-				version: Env.get('VERSION'),
-				environment: Env.get('NODE_ENV'),
-				context: Env.get('CONTEXT'),
+				name: env.get('NAME').asString(),
+				version: env.get('VERSION').toString(),
+				environment: env.get('NODE_ENV').toString(),
+				context: env.get('CONTEXT').toString(),
```

Comment:
why one time asString and 3 times toString?

Reply:
Autocomplete error. Will fix!